docs/manual/docs/cmd/spo/web/web-add.md

```diff
@@ -0,0 +1,64 @@
+# spo web add
+
+Create subsite
+
+## Usage
+
+```sh
+spo web add [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-t, --title [title]`|Subsite title
+`-d, --description [description]`|Subsite description, optional
+`-u, --webUrl [webUrl]`|Subsite relative url
+`-w, --webTemplate [webTemplate]`|Subsite template, eg. STS#0 (Classic team site)
```

Comment:
webTemplate is a required option and should be denoted using angle brackets, like `<webTemplate>`